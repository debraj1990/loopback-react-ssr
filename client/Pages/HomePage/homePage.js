import React from 'react';

export const Home = () => {
  return (
    <div className="center-align" style={{ marginTop: '200px' }}>
      <h3>Welcome</h3>
      <p>Check out these awesome features using Server Side Rendering</p>
    </div>
  );
};

//export default Home
export default {
  component: Home
};
