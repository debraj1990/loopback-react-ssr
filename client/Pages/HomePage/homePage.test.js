import React from 'react';
import {configure, shallow } from 'enzyme';
import toJson from 'enzyme-to-json'
import Adapter from 'enzyme-adapter-react-16';

import { Home } from './homePage';


configure({adapter: new Adapter });


describe('<Home />', () => {
    

    it('should render the HomePage', () => {
        const component = shallow(<Home />)
        const tree = toJson(component)
        expect(tree).toMatchSnapshot()
    })

    it('should contain h3 text Welcome', () => {
        const component = shallow(<Home />)
        expect(component.contains(<h3>Welcome</h3>)).toBeTruthy()
    })
})