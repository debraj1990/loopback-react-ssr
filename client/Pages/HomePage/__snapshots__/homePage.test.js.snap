// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render the HomePage 1`] = `
<div
  className="center-align"
  style={
    Object {
      "marginTop": "200px",
    }
  }
>
  <h3>
    Welcome
  </h3>
  <p>
    Check out these awesome features using Server Side Rendering
  </p>
</div>
`;
